lua/grapple-utils.lua

```diff
@@ -22,6 +13,55 @@ local function get_current_index(tags, current_path)
 	return nil
 end
 
+function M.cycle_to_index(opts)
```

Comment:
Actually I think just `move_to_index` was better. We can probably add a docstring describing the modulo behaviour later.

Reply:
Oh, actually 1-based indexing makes this a bit harder than just modulo.